### ISSUE TITLE
Batch size and learning rate

### ISSUE THREAD

**Issue by Myooooo** [avishkarsaha/translating-images-into-maps#11]
Hi. Thanks for the amazing work and contribution! 
I wonder why the batch size and learning rate are set differently in the source code from the paper. 
The paper stated: 
"_We train our network end-to-end with an Adam
optimizer, batch size 8 and initial learning rate of 5e−5,
which we decay by 0.99 every epoch for 40 epochs_"

But in the source code: 
`parser.add_argument("-l", "--lr", type=float, default=1e-4, help="learning rate")`
and
`
parser.add_argument(
        "-b", "--batch-size", type=int, default=1, help="mini-batch size for training"
    )
`
Thanks!

**Comment by Myooooo** [comment]
I can't reproduce the results from the paper using the default settings. Here are my outputs: 
train_ious.txt
`Epoch: 600, 
Total Loss: 41.64564514160156, 
s100_ious_per_class: [0.9633086  0.9630418  0.9567408  0.         0.8935271  0.95535564
 0.8352287  0.91729224 0.82856953 0.         0.94143116 0.772431
 0.6284842  0.9323648 ], 
s50_ious_per_class: [0.9690675  0.9660423  0.96897584 0.         0.8911782  0.95833045
 0.84155685 0.92406344 0.85713977 0.         0.94216543 0.7858166
 0.6800831  0.94098324], 
s25_ious_per_class: [0.9756959  0.96991634 0.97783333 0.         0.8899462  0.7916631
 0.83525336 0.9411748  0.85713774 0.         0.93781537 0.8334482
 0.7085929  0.9575054 ], 
s13_ious_per_class: [0.9790122  0.96631086 0.97900814 0.         0.8686852  0.7916619
 0.8753671  0.9117616  0.6428525  0.         0.9495894  0.910756
 0.8542234  0.9441347 ], 
s100_loss_per_class: [5.0171785e-02 7.5088745e-01 5.9365246e-02 6.3000000e+01 3.3000309e+00
 1.6906892e+00 1.9988866e-01 2.7971733e+00 8.1664543e+00 6.3000000e+01
 9.7722393e-01 5.6586081e-01 9.6497935e-01 3.9444879e-01], 
s50_loss_per_class: [2.4239659e-02 7.3934537e-01 1.8460825e-02 6.3000000e+01 3.2954257e+00
 1.6679329e+00 1.8942261e-01 2.7776437e+00 8.1452932e+00 6.3000000e+01
 9.6288759e-01 5.5086070e-01 9.1996020e-01 3.8559771e-01], 
s25_loss_per_class: [1.2026424e-02 7.3434430e-01 5.4079527e-03 6.3000000e+01 3.2944481e+00
 1.8340522e+00 1.9207710e-01 2.7670312e+00 8.1448488e+00 6.3000000e+01
 9.5948166e-01 5.1438648e-01 8.9774448e-01 3.7530217e-01], 
s13_loss_per_class: [5.8141286e-03 7.3392093e-01 3.4817569e-03 6.3000000e+01 3.3090463e+00
 1.8336529e+00 1.6512345e-01 2.7826216e+00 8.2862759e+00 6.3000000e+01
 9.5140320e-01 4.6182454e-01 7.9856169e-01 3.8186589e-01], `

val_ious.txt
`Epoch: 600,
Total Loss: 35.3160400390625,
s200_ious_per_class: [0.7004561  0.03919869 0.05355929 0.06599286 0.         0.
 0.00247366 0.         0.         0.         0.         0.
 0.         0.        ], 
s100_ious_per_class: [0.6947752  0.04115277 0.05556437 0.07176002 0.         0.
 0.00279711 0.         0.         0.         0.         0.
 0.         0.        ], 
s50_ious_per_class: [0.69750965 0.05351232 0.08032346 0.04858857 0.         0.
 0.00545815 0.         0.         0.         0.         0.
 0.         0.        ], 
s25_ious_per_class: [0.6985369  0.07322484 0.14161275 0.09709989 0.         0.
 0.01690527 0.         0.         0.         0.         0.
 0.         0.        ], 
s13_ious_per_class: [0.72845477 0.11243327 0.23103584 0.16527076 0.         0.
 0.05965656 0.         0.         0.         0.         0.00833332
 0.04761888 0.        ], 
s200_loss_per_class: [ 0.23925553  0.8731207   0.8439295   0.8874003  15.          2.4999993
  0.93309706 15.         15.         15.         15.          0.9374987
  1.875      15.        ], 
s100_loss_per_class: [ 0.22965674  0.8689645   0.8411137   0.8803522  15.          2.499999
  0.93188447 15.         15.         15.         15.          0.937498
  1.875      15.        ], 
s50_loss_per_class: [ 0.23641275  0.85603386  0.79646754  0.87034154 15.          2.4999986
  0.9283314  15.         15.         15.         15.          0.9372381
  1.8749999  15.        ], 
s25_loss_per_class: [ 0.23035507  0.8257067   0.70719624  0.8509307  15.          2.4999897
  0.9046947  15.         15.         15.         15.          0.9374631
  1.8749864  15.        ], 
s13_loss_per_class: [ 0.20534727  0.7836538   0.59846616  0.7804562  15.          2.4999874
  0.8391777  15.         15.         15.         15.          0.9136455
  1.8189114  15.        ], `

**Comment by wang-TJ-20** [comment]
@Myooooo I have the same problem as you

**Comment by Myooooo** [comment]
> @Myooooo I have the same problem as you

Thanks for your reply. I think it's because the dataset provided was simplified with fewer samples to reduce training time for personal computers.

**Comment by wang-TJ-20** [comment]
I think so too. Now i don't know how to generate&nbsp;the ground truth maps.&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "avishkarsaha/translating-images-into-maps"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年8月18日(星期四) 下午3:20
***@***.***&gt;;
抄送:&nbsp;" ***@***.******@***.***&gt;;
主题:&nbsp;Re: [avishkarsaha/translating-images-into-maps] Batch size and learning rate (Issue #11)





  
@Myooooo I have the same problem as you
  
Thanks for your reply. I think it's because the dataset provided was simplified with fewer samples to reduce training time for personal computers.

**Comment by Handsomer1999** [comment]
@Myooooo I have the same problem as you
